pkg/apis/build/v1alpha2/build_types.go

```diff
@@ -56,6 +56,7 @@ type BuildSpec struct {
 	ServiceAccountName string                        `json:"serviceAccountName,omitempty"`
 	Source             corev1alpha1.SourceConfig     `json:"source"`
 	Cache              *BuildCacheConfig             `json:"cache,omitempty"`
+	RunImage           string                        `json:"runImage,omitempty"`
```

Comment:
To line up with our other crds, maybe this can be a struct with a nested `image` field like we have in the [clusterstack spec](https://github.com/pivotal/kpack/blob/33d72b29eed4afa7ce4830783e4b6ac24deddbbb/pkg/apis/build/v1alpha2/cluster_stack_types.go#L32-L33) and the [clusterstore spec](https://github.com/pivotal/kpack/blob/33d72b29eed4afa7ce4830783e4b6ac24deddbbb/pkg/apis/build/v1alpha2/cluster_store_types.go#L32)

Reply:
Yes! All images in kpack spec fields should have the key `image`. I think @tylerphelan suggested maybe putting the run image it in the builder spec BuildBuilderSpec?

Reply:
If I remember from the meeting last Friday, the current BuildBuilderSpec has the following structure:

```golang
type BuildBuilderSpec struct {
	Image string `json:"image,omitempty"`
	// +patchMergeKey=name
	// +patchStrategy=merge
	// +listType
	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,15,rep,name=imagePullSecrets"`
}
```

We already have an `Image` field there, adding it there requires to refactor a little bit that structure, right? and I think that's a change that can make some troubles to current users.

Reply:
I like:
`Build.spec.runImage.image`